Themes/default/languages/Install.english.php

```diff
@@ -371,6 +371,11 @@
 $txt['warning_lang_old'] = 'The language files for your selected language, %1$s, have not been updated to the latest version. Upgrade will continue with the forum default, %2$s.';
 $txt['warning_lang_missing'] = 'The upgrader could not find the &quot;Install&quot; language file for your selected language, %1$s. Upgrade will continue with the forum default, %2$s.';
 
+// Attachment & Avatar folder checks
+$txt['warning_av_missing'] = 'Warning!  Avatar directory not found.  Continuing may be unsafe.  Please confirm folder settings before proceeding.';
+$txt['warning_custom_av_missing'] = 'Warning!  Custom avatar directory not found.  Continuing may be unsafe.  Please confirm folder settings before proceeding.';
+$txt['warning_att_dir_missing'] = 'Warning!  One or more attachment directories not found.  Continuing may be unsafe.  Please confirm folder settings before proceeding.';
```

Comment:
Are the double spaces intended?

Reply:
YES THEY ARE NECESSARY!!!!

OK, I removed 'em...   You young'uns...